doc/users/next_whats_new/2017_12_17_compare_colors.rst

```diff
@@ -0,0 +1,4 @@
+Comparision of 2 colors in Matplotlib
+-------------------------------------
+
+As the colors in Matplotlib can be specified with a wide variety of ways, this new version consists of a method ``matplotlib.colors.same_color`` which allows 2 legend colors to be compared.
```

Comment:
Just use single ticks so this generates a link.  

The English is a bit convoluted..:

```
New method `matplotlib.colors.same_color` checks if  two `~matplotlib.colors` are the same.  
```